cdci_data_analysis/analysis/parameters.py

```diff
@@ -421,28 +460,48 @@ def check_int_value(value, units=None,name=None):
                 raise RuntimeError('type of ', name, 'not valid', type(value))
 
 
-
-
-
 class Time(Parameter):
-    def __init__(self,value=None,T_format=None,name=None,Time_format_name=None):
+    def __init__(self, value=None, T_format='isot', name=None, Time_format_name=None):
 
         #_allowed_units = astropyTime.FORMATS
 
         #wtform_dict = {'iso': StringField}
         #wtform_dict['mjd'] = FloatField
         #wtform_dict['prod_list'] = TextAreaField
 
-        super(Time,self).__init__(value=value,
-                                  units=T_format,
-                                  units_name=Time_format_name,
-                                  name=name,
-                                  allowed_units=None)
+        super(Time, self).__init__(value=value,
+                                   units=T_format,
+                                   units_name=Time_format_name,
+                                   default_units='isot',
+                                   name=name,
+                                   allowed_units=None)
                                   #wtform_dict=wtform_dict)
 
-
         self._set_time(value,format=T_format)
 
+    @staticmethod
+    def set_group_par(par_group, par_dic, params_not_to_be_included=None, verbose=False):
+        if params_not_to_be_included is None:
+            params_not_to_be_included = []
+        list_par_group = list(par_group)
+        for par in list_par_group:
+            if par.name is not None and par.name not in params_not_to_be_included:
+                par.set_from_form(par_dic, verbose=verbose)
+            logger.info("set_pars_from_dic>> par: %s par.name: %s par.value: %s par_dic[par.name]: %s",
+                        par, par.name, par.value, par_dic.get(par.name, None))
+
+        # set the default, isot
+        for par in list_par_group:
+            par_dic[par.units_name] = 'isot'
```

Comment:
this `"isot"` value is stored in `default_units`. And if that's used, the method doing this can be in Parameter (except maybe if possible to avoid new `set_group_par` entirely, I do not see why not, since it does not count anything anymore).

Reply:
It is now indeed like that, and there is no longer `set_group_par`, since it is not needed.